Custom App Creation Failing

Doing step by step instructions here:
https://docs.commercetools.com/custom-applications/getting-started/installing-a-starter-application

OS: Ubuntu 18.04.4 LTS
OS type: 64-bit
node: v12.16.3
npm: 6.14.4

After npm start it throws error:
`./node_modules/@commercetools-frontend/application-shell/dist/application-shell-index.es.jsModule not found: Can't resolve 'apollo-client/version' in '/dist-to-my-project/v2/custom-order-app/node_modules/@commercetools-frontend/application-shell/dist'`

Steps taken until error:
1. npx @commercetools-frontend/create-mc-app \ my-new-custom-application-project \ --template starter (its successful installed)
2. cd my-new-custom-application-project
3. npm start
4. Above Error Message received

--- a/.changeset/soft-beans-deliver.md
+++ b/.changeset/soft-beans-deliver.md
@@ -1,0 +1,5 @@
+---
+'merchant-center-application-template-starter': patch
+---
+
+Add apollo-client as dependency as it is peer dependency or react-apollo.

--- a/.changeset/soft-beans-deliver.md
+++ b/.changeset/soft-beans-deliver.md
@@ -2,4 +2,4 @@
 'merchant-center-application-template-starter': patch
 ---
 
-Add apollo-client as dependency as it is peer dependency or react-apollo.
+Add apollo-client as dependency as it is peer dependency of react-apollo.

--- a/application-templates/starter/package.json
+++ b/application-templates/starter/package.json
@@ -23,6 +23,7 @@
     "@commercetools-uikit/icons": "10.18.4",
     "@commercetools-uikit/spacings": "10.18.4",
     "@commercetools-uikit/text": "10.18.4",
+    "apollo-client": "2.6.8",
     "prop-types": "15.7.2",
     "react": "16.13.1",
     "react-apollo": "3.1.5",

